docs/connector-development/cdk-python/resumable-full-refresh-stream.md

```diff
@@ -0,0 +1,77 @@
+# Resumable Full Refresh Streams
```

Comment:
@girarda fyi i added a docs for RFR instead of inlining comments for certain methods etc. I still need to proof grammar and clean it up a little but just a heads up since some PR comments called this out